package.json

```diff
@@ -37,7 +37,7 @@
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "pretest": "npm run lint",
     "test": "npm run jest",
-    "jest": "cross-env LOG_LEVEL=error MOCK_REDIS=1 jest --",
+    "jest": "cross-env LOG_LEVEL=error MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest --",
```

Comment:
Timing related stuff in tests makes me nervous, since it can fail in ways you don't anticipate under various conditions.  Can we use https://jestjs.io/docs/en/timer-mocks to mock the timing?

Reply:
Based on conversation, let's take this now, and just watch how it performs, and we can circle back if we get nervous later.